equinix/resource_network_device.go

```diff
@@ -414,6 +418,20 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 			ValidateFunc: validation.IsUUID,
 			Description:  neDeviceDescriptions["ProjectID"],
 		},
+		neDeviceSchemaNames["DiverseFromDeviceUUID"]: {
+			Type:          schema.TypeString,
+			Computed:      true,
+			Optional:      true,
+			ValidateFunc:  validation.IsUUID,
+			ConflictsWith: []string{neDeviceSchemaNames["Secondary"]},
+			Description:   neDeviceDescriptions["DiverseFromDeviceUUID"],
+		},
+		neDeviceSchemaNames["DiverseFromDeviceName"]: {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Optional:    true,
+			Description: neDeviceDescriptions["DiverseFromDeviceName"],
```

Comment:
Does the device name diversity option also conflict with Secondary, as the UUID option does?

Reply:
no @displague , only diverse device uuid is being passed in the payload. diverse device name is present only in device details response(data source - network device)